examples/00-fluent/frozen_rotor_workflow.py

```diff
@@ -0,0 +1,424 @@
+# steady-state flows, where the rotor's motion is constant.
+# By using this approach, engineers can obtain accurate predictions of the
+# flow field and performance characteristics of rotating machinery without the need for
+# complex and computationally expensive simulations.
+#
+#
+# .. image:: ../../_static/pump-volute-geom.png
+#    :align: center
+#    :alt: Pump Volute Geometry View
+
+################################################################################################################
+# Outline
+# ==============================================================================================================
+
+# The example demonstrates the following:
+
+# *Overview & Problem description
+# *Launching Fluent in solver mode.
+# *Downloading a mesh files from the examples repository.
+# *Initial setup
+# *Mesh configuration
+# *Model selection & Material definition
+# *Defining cell zone conditions & boundary conditions
+# *Turbomachinery configuration
+# *Solver settings
+# *Report definitions
+# *Initialization
+# *Running the simulation
+# *Post-processing the results
+# *Visualizing the results
+# *Saving the case file
+# *Closing the solver
+
+
+################################################################################################################
+# Import required libraries/modules
+# ==============================================================================================================
+import math
+
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+
+################################################################################################################
+# Download the mesh file
+# ==============================================================================================================
+
+# Download the mesh file from the examples repository
+# Impeller and volute mesh files
+impeller_mesh = examples.download_file(
+    "impeller.msh.h5",
+    "pyfluent/examples/pump-volute",
+)
+
+volute_mesh = examples.download_file(
+    "volute.msh.h5",
+    "pyfluent/examples/pump-volute",
+)
+
+################################################################################################################
+# Define Constants
+# ==============================================================================================================
+
+rho_water = 998.2  # kg/m^3
+mu_water = 0.001002  # kg/(m.s)
+g = 9.81  # m/s^2
+# Impeller speed
+impeller_speed = 1450  # rpm
+# Convert to rad/s
+impeller_speed_rad = impeller_speed * 2 * math.pi / 60  # rad/s
+
+################################################################################################################
+# Launch Fluent with solver mode and print Fluent version
+# ==============================================================================================================
+
+solver = pyfluent.launch_fluent(
+    mode="solver",
+    ui_mode="gui",
```

Comment:
```suggestion
    mode=pyfluent.FluentMode.SOLVER,
    ui_mode=pyfluent.UIMode.GUI,
```
I think that's correct.